src/QsCompiler/RoslynWrapper/ExpressionStatement.fs

```diff
@@ -164,6 +164,9 @@ module Expressions =
 
     // left >>> right
     let (.>>>.) left right =
+        // TODO: Should be replaced with a use of SyntaxFactor.Token to generate an unsigned shift right. For reference,
```

Comment:
```suggestion
        // TODO: Should be replaced with a use of SyntaxFactory.Token to generate an unsigned shift right. For reference,
```

Reply:
I'll fix this and add a link to the Rosyln issue.